### ISSUE TITLE
Unable to install other wow versions of little wigs

### ISSUE THREAD

**Issue by Raina1007** [WowUp/WowUp#1283]
**Describe the bug**
When installing from URL, not able to install other versions of little wigs beyond what is latest. Unfortunately they don't or can't do 1 release for each flavor of wow. Probably due to how much data it would pull for each flavor of wow for the dungeons. So depending on which one is latest is what you get even if its not what you wanted.

**To Reproduce**
1. Go to install from URL in get addons
2. Click on click the button and paste in this URL: https://github.com/BigWigsMods/LittleWigs
3. click import then get pop up about wrong version and install at own risk, then click install
4. See error - see my addons to see that you installed retail version at this time

**Expected behavior**
To install correct flavor game version of little wigs 

**Screenshots**
![Screen Shot 2022-07-27 at 6 30 28 PM](https://user-images.githubusercontent.com/72997449/181383636-1df07101-7862-4151-b6a0-ca703345ab7d.png)

**Desktop (please complete the following information):**
I'm on v2.9.0 - beta 2, but person you reported it in help me is probably on v2.8.3

**Additional context**
Basically try to install little wigs for tbc, you will get retail version instead. It does have a release.json so maybe a work around can be figured out using that
